### DISCUSSION TITLE
Simplifying Updates After Renovate/Dependabot

### DISCUSSION THREAD

**Discussion by sheldonhull** [miniscruff/changie#601] [Show and tell]
Decided to share after reading some of the posts from pulumi team. I've been trying to wrap changie workflow commands so mostly folks just need to run changie new.

I tried to eliminate the need to run changie cli so I could rebuild and release after running renovate updates. Would have preferred to write some dagger mage code but needed a quick way to help team release dependency updates or new go versions. 

[changie-trigger-release.yml](https://github.com/DelineaXPM/github-workflows/blob/ae8b9693f280e7e9e1213f10bd4535ff12b194f8/.github/workflows/changie-trigger-release.yml)

Maybe will be a little inspiring. 

Workflow can be challenging with changie as it's non-opinionated on this but also probably why it's gaining more popularity. 😉

Long term I might feed the git diff into charms mods cli and generate a change entry based on OpenAI analysis of the git diff but worst case I'll probably just filter from last tag all the chore(deps) and for each add a changie entry.

**Comment by miniscruff** [comment]
This looks interesting, I will try and find a place to put this sort of stuff on the website when I moved to mkdocs finally ( need to rework some of my codegen for it ). I played around with getting `/changie do stuff` comments on PRs to make updates to them directly but could not seem to get the branching stuff working properly. But that is another thing I would like to include.
